editors/vscode/package.json

```diff
@@ -188,10 +188,10 @@
     "vscode:prepublish": "pnpm build:wasm_lsp-release && pnpm compile && shx echo \"GPL-3.0-only OR LicenseRef-Slint-Software-3.0\" > LICENSE.txt",
     "build:lsp": "cargo build -p slint-lsp",
     "build:lsp-release": "cargo build --release -p slint-lsp",
-    "build:wasm_lsp": "env-var wasm-pack build --dev     --target web --no-pack ../../tools/lsp --out-dir {{npm_config_local_prefix}}/out -- --no-default-features --features backend-winit,renderer-femtovg,preview",
-    "build:wasm_lsp-release": "env-var wasm-pack build --release --target web --no-pack ../../tools/lsp --out-dir {{npm_config_local_prefix}}/out -- --no-default-features --features backend-winit,renderer-femtovg,preview",
+    "build:wasm_lsp": "env-var wasm-pack build --dev     --target web --no-pack ../../tools/lsp --out-dir $(pwd)/out -- --no-default-features --features backend-winit,renderer-femtovg,preview",
+    "build:wasm_lsp-release": "env-var wasm-pack build --release --target web --no-pack ../../tools/lsp --out-dir $(pwd)/out -- --no-default-features --features backend-winit,renderer-femtovg,preview",
     "compile": "node ./esbuild.js",
-    "local-package": "shx mkdir -p bin && shx cp ../../target/debug/slint-lsp* bin/ && npx vsce package",
+    "local-package": "shx mkdir -p bin && shx cp ../../target/debug/slint-lsp* bin/ && pnpm dlx vsce package",
```

Comment:
Hm, this works for us running `local-package`, but I think the invocation vsce when publishing to the store (using HaaLeo/publish-vscode-extension@v1 ) will still call npm.

Reply:
Maybe building the vsix manually in the CI and setting `extensionFile` could work?